.github/actions/clp-core-build-containers/action.yaml

```diff
@@ -2,6 +2,9 @@ name: "clp-core-build-containers"
 description: "Builds and publishes container images for CLP-core's dependencies"
 
 inputs:
+  image_name:
+    description: "Name of the image (without registry or repository) to create"
+    required: true
   os_name:
     description: "Name of container OS"
     required: true
```

Comment:
Do you think its better to just pass in clp-env-base-${{inputs.os_name}} instead of just os_name so its more consisten with image_name?

Reply:
Yeah, true, we could.